facebook/OpenBIC
# fby4: ff: Disable DIMM monitor

Description
- Remove the wrong way to monitor DIMM until the right way is discussed.

Motivation
- Avoid a lot of error messages affecting the BIC console.

Test plan
- Build code: Pass
- Check BIC console: Pass

Log
- Before [00:05:01.793,000] <inf> plat_power_seq: CXL power on success [00:05:01.793,000] <wrn> power_status: DC_STATUS: on uart:~$
uart:~$ [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x0 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x1 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x2 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x3 of thread 4, status0x4 [00:05:08.077,000] <err> pldm_sensor: Failed to read sensor_num 0x0 of thread 4, status0x4 [00:05:08.077,000] <err> pldm_sensor: Failed to read sensor_num 0x1 of thread 4, status0x4 [00:05:08.077,000] <err> pldm_sensor: Failed to read sensor_num 0x2 of thread 4, status0x4 [00:05:08.077,000] <err> pldm_sensor: Failed to read sensor_num 0x3 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x0 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x1 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x2 of thread 4, status0x4 [00:05:07.077,000] <err> pldm_sensor: Failed to read sensor_num 0x3 of thread 4, status0x4

- After [00:04:09.795,000] <inf> plat_power_seq: CXL power on success [00:04:09.795,000] <wrn> power_status: DC_STATUS: on uart:~$
uart:~$
uart:~$
